dagshub/data_engine/model/datapoint.py

```diff
@@ -45,8 +45,15 @@ def __getitem__(self, item):
     def __setitem__(self, key, value):
         self.datasource.implicit_update_context.update_metadata(self.path, {key: value})
 
-    def delete_metadata(self, name):
-        self.datasource.delete_metadata(self.path, self.datapoint_id, name)
+    def delete_metadata_field(self, name):
+        """
+        delete a metadata field from this datapoint
+        the deleted value can be accessed using versioned query
+        with time before the deletion
+        Args:
+             name - name of field
```

Comment:
Change arg name from name -> field

```suggestion
        Delete a metadata field from this datapoint.
        The deleted value can be accessed using versioned query
        with time set before the deletion.
        
        Args:
             field - field to delete
```